src/ScriptEditor/ui/ScriptEditorRoot.tsx

```diff
@@ -893,7 +895,11 @@ export function Root(props: IProps): React.ReactElement {
         </Box>
         <OptionsModal
           open={optionsOpen}
-          onClose={() => setOptionsOpen(false)}
+          onClose={() => {
+            sanitizeTheme(Settings.EditorTheme);
```

Comment:
should we be saving it when we close?

Reply:
I think it's fine to save on close, since the user already clicks "save" within the theme editor modal, so it would be odd for the changes to not be applied.

Beyond that, the theme is saved in the save object when the theme is saved from within the editor modal, so not applying it on close would lead to an odd situation where the theme is applied next time the editor is loaded.